test/integration/test_language_container_deployer.py

```diff
@@ -52,14 +58,38 @@ def test_language_container_deployer(
         assert_udf_running(pyexasol_connection, TEST_LANGUAGE_ALIAS, TEST_SCHEMA)
 
 
-def test_language_container_deployer_alter_session(
+def test_cert_failure(itde: config.TestConfig):
+    """
+    Verifies that connecting with an invalid SSL certificate fails with an
+    exception.
+    """
+    parsed_url = urlparse(itde.bucketfs.url)
+    with pytest.raises(ExaConnectionFailedError, match="[SSL: CERTIFICATE_VERIFY_FAILED]"):
+        deployer = LanguageContainerDeployer.create(
+            language_alias=TEST_LANGUAGE_ALIAS,
+            dsn=f"{itde.db.host}:{itde.db.port}",
+            db_user=itde.db.username,
+            db_password=itde.db.password,
+            bucketfs_name="bfsdefault",
+            bucketfs_host=parsed_url.hostname,
+            bucketfs_port=parsed_url.port,
+            bucketfs_use_https=False,
+            bucketfs_user=itde.bucketfs.username,
+            bucketfs_password=itde.bucketfs.password,
+            bucket="default",
+            use_ssl_cert_validation=True,
+            path_in_bucket="container",
+            )
+
+def test_download_and_alter_session(
```

Comment:
The comment says "...uploading the container..." but the test name ist `test_download...` ?

Reply:
Uh - Oh. Yes, I understand. 
The test basically existed before.
The docstring ended in "This test also covers downloading a container file from a URL."

Actually it's 3 steps I think. I updated the docstring in the next push:
>  Tests the deployment of a container in 3 stages - 1. download a container file from a URL, 2. upload the file to the BucketFS and 3. activate it at the Session level.